plugin/pkg/scheduler/algorithm/priorities/util/topologies.go

```diff
@@ -52,8 +52,12 @@ func PodMatchesTermsNamespaceAndSelector(pod *v1.Pod, affinityPod *v1.Pod, term
 	return true, nil
 }
 
-// nodesHaveSameTopologyKeyInternal checks if nodeA and nodeB have same label value with given topologyKey as label key.
-func nodesHaveSameTopologyKeyInternal(nodeA, nodeB *v1.Node, topologyKey string) bool {
+// NodesHaveSameTopologyKey checks if nodeA and nodeB have same label value with given topologyKey as label key.
+// Returns false if topologyKey is empty.
+func NodesHaveSameTopologyKey(nodeA, nodeB *v1.Node, topologyKey string) bool {
+	if len(topologyKey) == 0 {
```

Comment:
The [comment](https://github.com/kubernetes/kubernetes/blob/master/pkg/api/v1/types.go) for TopologyKey says

```
For PreferredDuringScheduling pod anti-affinity, empty topologyKey is interpreted
as "all topologies ("all topologies" here means all the topologyKeys indicated by
scheduler command-line argument --failure-domains)
```

I think we want to continue to support this behavior.